ingester/download_ocr_models.py

```diff
@@ -0,0 +1,3 @@
+from unstructured_inference.models.tables import load_agent
+
+load_agent()
```

Comment:
unlikely to be a problem i know, but a good idea to use the usual:

```python
if __name__ == "__main__":
    load_agent()
```

just in case anyway tries to import this file

Reply:
Added, great shout!